environment.yml

```diff
@@ -4,9 +4,5 @@ channels:
   - default
 
 dependencies:
-  - ale>=0.8.8
   - cmake>=3.15
-  - csm
-  - eigen
-  - nlohmann_json
-  - proj
+  - sqlite
```

Comment:
This is concerning because many packages depend upon sqlite. For example:

- [ASP](https://github.com/NeoGeographyToolkit/StereoPipeline/blob/master/conda/asp_3.2.0_linux_env.yaml#L232)
- [GDAL](https://github.com/conda-forge/gdal-feedstock/blob/main/recipe/meta.yaml)
- SocetGXP???

Does the usgscsm depend on sqlite or is this because of the projection stuff inside the CSM?

Reply:
I think our last test build also needed the **curl** lib, unless that was removed from the build. 
https://github.com/OSGeo/PROJ/blob/a654e42074a167a03e1e3ff7c2009c04502830bf/docs/source/install.rst#build-requirements 

I can currently work-around the extra libs for SocetGXP, but it is not graceful. Ideally, bundling proj with usgscsm can be maybe optional (at least for Windows) but I'm not sure how best to resolve this.